README.md

```diff
@@ -1,2 +1,118 @@
+
+The list of recognised apps is:
+
+* `Security`
+* `Payment`
+* `Pipeline`
+* `TransferToPensionAge`
+* `MI12CaseControl`
+* `QLR`
+* `ReleaseComparison`
+
+### Guard duty notifications
+
+If the message has the `detail-type` field and the field contains `GuardDuty Finding` it is a guard duty notification. If this is the case then a notification is logged with the text `AWS GuardDuty Finding Type` and a URL is consructed to go directly to the finding in the AWS console.
+
+### Configuration notifications
+
+If the message has the `configRuleName` field it is a configuration notification. If this is the case then a notification is logged with the text `AWS Config Compliance Change detected` and a URL is consructed to go directly to the finding in the AWS console.
+
+### Cloudwatch event notifications
+
+If the message has the `messageType` field it is a cloudwatch event notification. If this is the case then a notification is logged with the text `AWS Config CloudWatch Event` and a URL is consructed to go directly to the finding in the AWS console. Details of the event are pulled out and provided as extra information in the slack message.
+
+### Cloudwatch alarm notifications
+
+If the message has the `AlarmName` field it is a cloudwatch alarm notification. If this is the case their are two sub scenarios based on the message itself.
+
+#### Prowler alarms
+
+If the message has the `Namespace` field and its value is set to `Prowler/Monitoring` then the message is assumed to be from the Prowler monitoring that is set up. In this case, the message is processed with a title of the name of the monitoring alert triggered an a URL to go to the alert directly is generated and set in the slack message.
+
+#### Custom alarms
+
+Without the `Namespace` field or if it is set to a different value, then the message is treated differently. The title will be set as the following:
+
+* `{icon} *{environment_name.upper()}*: "_{alarm_name}_" in {region}'`
+
+The `region` and `alarm_name` fields come directly from the message and `environment_name` is from the `AWS_ENVIRONMENT_NAME` environment variable.
+
+The icon is set from the severity and the type of notification. These values also set the channel that the slack notification goes to and they are logged in the message as well. Their values are retrieved from the tags on the cloudwatch alarm itself and are detailed in the section below.
+
+#### Custom alarm tags
+
+The following tags are retrieved from the cloudwatch alarm and each one has a default in case it is not present:
+
+* `severity` -> this is used to determine the icon used for the alarm and the channel it goes to, supported values are `Low`, `Medium`, `High` and `Critical`. Default is `Medium`
+
+* `notification_type` -> this is used to determine the icon used for the alarm and the channel it goes to, supported values are `Information`, `Warning` and `Error`. Default is `Warning`
```

Comment:
Descriptions for both `severity` and `notification_type` are the same but it's not clear how they are different / why both exist if they are the same

Reply:
I've added a bit saying it's a combination of both that is used. The combinations are details in the `Environment variables` section so have pointed it to there too.